presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -1213,7 +1214,7 @@ private void printAssignments(Assignments assignments, int indent)
 
         private String formatOutputs(Iterable<Symbol> symbols)
         {
-            return Joiner.on(", ").join(Iterables.transform(symbols, input -> input + ":" + types.get(input).getDisplayName()));
+            return Joiner.on(", ").join(Iterables.transform(symbols, input -> input + ":" + types.getOrDefault(input, UNKNOWN).getDisplayName()));
```

Comment:
+1 Investigate whether this is not a bug somewhere else in the code that the type is missing for symbol. I wouldn't assume that is is `UNKNOWN` by default.